docs/index.html

```diff
@@ -108,36 +109,31 @@ <h1>JsQRScanner example</h1>
       </div>
       <br>
     </div>
-    <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity="sha256-4+XzXVhsDmqanXGHaHvgh1gMQKX40OUvDEBTu8JcmNs=" crossorigin="anonymous"></script>
-    <script>
 
-    </script>
   <script type="text/javascript">
-    function onQRCodeScanned(scannedText)
-    {
-      const data = { scannedQrCode: scannedText}
-      fetch('https://demandid-attendance.herokuapp.com/api/member/details', {
-        method: 'POST',
-        headers: {
-          'Content-Type': 'application/json',
-        },
-        body: JSON.stringify(data)
-      })
-      .then(response => response.json())
-      .catch((error) => {
-        console.error('Error:', error)
+    function requestDetailsFromApi(scannedText) {
+    $.ajax({
+      type: "GET",
+      url: "https://demandid-attendance.herokuapp.com/api/member/details",
```

Comment:
I think this might not be the best.
Is the API supposed to be available to the general public?🤔